arangod/IResearch/IResearchView.cpp

```diff
@@ -275,7 +275,12 @@ inline void insertDocument(
 
   // User fields
   while (body.valid()) {
-    doc.insert(irs::action::index_store, field);
+    if (ValueStorage::NONE == field._storeValues) {
+      doc.insert(irs::action::index, field);
+    } else {
```

Comment:
the `else` case covers types `ID` and `FULL`, but is identical for both. Is it intentional?

Reply:
`FULL` is not supported at the moment and meant to be treated as the reserved value for future usage

Reply:
ah, ok!